lib/portlayer/event/collector/vsphere/collector.go

```diff
@@ -118,7 +118,7 @@ func (ec *EventCollector) Start() error {
 	ctx := context.Background()
 
 	// events per page
-	pageSize := int32(1)
+	pageSize := int32(25)
```

Comment:
suggest more comment about why this page size - git blame could identify this issue, but there's ways that can get mixed up.